Tests/Marketplace/upload_packs.py

```diff
@@ -628,6 +669,49 @@ def option_handler():
     return parser.parse_args()
 
 
+def add_pr_comment(comment):
+    """Add comment to the pull request.
+
+    Args:
+        comment (string): The comment text.
+
+    """
+    token = os.environ['CONTENT_GITHUB_TOKEN']
+    branch_name = os.environ['CIRCLE_BRANCH']
+    sha1 = os.environ['CIRCLE_SHA1']
+
+    print('branch_name = ' + branch_name)
+    print('sha1 = ' + sha1)
+
+    query = '?q={}+repo:demisto/content+org:demisto+is:pr+is:open+head:{}+is:open'.format(sha1, branch_name)
```

Comment:
use f-strings
if you stated `repo`, is `org` required?
```suggestion
    query = f'?q={sha1}+repo:demisto/content+org:demisto+is:pr+is:open+head:{branch_name}'
```

Reply:
org is not required, i removed it